style.css

```diff
@@ -382,3 +374,223 @@ form textarea {
   display: flex;
   justify-content: center;
 }
+
+.btn-transition:hover {
+  transition: width 2s, height 2s, background-color 2s, transform 2s;
+  background: #6070ff;
+  color: white;
+  width: 30%;
+}
+
+nav {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
- [ ] **OPTIONAL**
It seems that your mobile version got a little overflow after you created the desktop version. Kindly check on that!
![Screenshot](https://user-images.githubusercontent.com/47625407/121763401-1bcf9f80-cb3c-11eb-9e31-6faf50f5eb2e.png)

> Please keep in mind that this is a mandatory change and I'm sorry it didn't show up on the first review.